apim-apk-agent/internal/synchronizer/apis_fetcher.go

```diff
@@ -151,6 +154,44 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUIDList []string, k8sClient clien
 				// Todo: Read the apis.zip and extract the api.zip,deployments.json
 			}
 
+			if err != nil {
+				logger.LoggerMsg.Error("Error while reading zip", err)
+			}
+
+			artifact, decodingError := transformer.DecodeAPIArtifact(data.Resp)
+
+			if decodingError != nil {
+				logger.LoggerSync.Infof("Error while decoding the API Project Artifact")
+				break
+			}
+
+			apkConf, _, apkErr := transformer.GenerateAPKConf(artifact.APIJson)
+			logger.LoggerSync.Infof(apkConf)
+
+			if apkErr != nil {
+				log.Println("Error in generating APK conf")
```

Comment:
let's use proper loggers in all the such places

Reply:
Done!